packages/mermaid/src/docs/README.md

```diff
@@ -263,7 +263,7 @@ Update version number in `package.json`.
 npm publish
 ```
 
-The above command generates files into the `dist` folder and publishes them to npmjs.org.
+The above command generates files into the `dist` folder and publishes them to <npmjs.org>.
```

Comment:
Why is `<>` added?

Reply:
Since `npmjs` isn't a recognized word, and the file is markdown, I used `<npmjs.org>` to turn it into a link within the markdown. I could have turn it into ``npmjs.org``, but it seemed more fitting to make it a link.

Reply:
I thought that might be the case, but GitHub doesn't render it as a link. 

<img width="756" alt="image" src="https://user-images.githubusercontent.com/10703445/194683154-ee9f4a20-7024-414f-8019-e96da9a744c3.png">